src/components/projects/wbs-details/work-package-container/changes-list/changes-list.tsx

```diff
@@ -20,7 +20,8 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
       headerRight={<></>}
       list={changes.map((ic) => (
         <>
-          [<Link to={`${routes.CHANGE_REQUESTS}/${ic.crId}`}>#{ic.crId}</Link>] {ic.detail}
+          [<Link to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>#{ic.changeRequestId}</Link>
+          ] {ic.detail}
```

Comment:
The formatting of the code here looks a little funky. Is it what prettier did? Or can it be adjusted with some manual formatting, like split the two tags for the link element onto their own lines. And if we do manual formatting does it accidentally change the display on the front-end or no?

Reply:
It is what prettier did, and does not change any of the display on the front end that I could tell. I tried a few manual changes with more new lines and it still did not change the display unless I added non-whitespace characters. Did you want something like this with many shorter lines? I'm not sure if the indentation is correct since all prettier would do given the current settings is revert it to what it currently is.
<img width="495" alt="Screen Shot 2021-10-04 at 9 10 36 PM" src="https://user-images.githubusercontent.com/51572473/135945273-95e57208-0b0b-40a3-bc79-9b8f343e7ac9.png">